### ISSUE TITLE
Incorrect warning for aria-expanded on role combobox

### ISSUE THREAD

**Issue by BerndWessels** [ThePacielloGroup/ARC-Tracker#8]
### Summary

The Arc Toolkit displays a warning for elements with `role="combobox"` and `aria-expanded="true"` combination.

But this combination is absolutely compliant to the standard and even recommended with examples in the official documentation https://www.w3.org/TR/wai-aria-1.2/#combobox

### Expected result

No warning

### Actual result

A warning is shown: `ariaExpandedOnInvalidRole - Warning: aria-expanded is on an element where it will not be conveyed to screen reader users - Recommend: aria-expanded should be placed on the element which controls the visibility of another element`

### Test Case

 Official example from here https://www.w3.org/TR/wai-aria-1.2/#combobox

```html
<label for="tag_combo">Tag</label>
  <input type="text" id="tag_combo"
      role="combobox" aria-autocomplete="list"
      aria-haspopup="listbox" aria-expanded="true"
      aria-controls="popup_listbox" aria-activedescendant="selected_option">
<ul role="listbox" id="popup_listbox">
   <li role="option">Zebra</li>
   <li role="option" id="selected_option">Zoom</li>
</ul>
```

#### ARC Toolkit version

3.3.2.0

#### Operating System and version

Windows 10

#### Browser and version:

Chrome Version 84.0.4147.135 (Official Build) (64-bit)

**Comment by stevefaulkner** [comment]
@BerndWessels Thanks for the bug report

**Comment by ferllings** [comment]
Hi @BerndWessels,

I can't reproduce with 3.3.2. 
Could you try with https://cdpn.io/TPG/debug/98ebf17606ff880f8b436428ae3401cd

**Comment by BerndWessels** [comment]
@ferllings @stevefaulkner Hi guys, thanks for the response.

After a bit more digging it turns out that the warning shows up when the `<input>` does not have an `type` attribute.

![image](https://user-images.githubusercontent.com/1776695/91546427-867d6900-e976-11ea-9afa-7f7fd129a7be.png)

With the `type="text"` attribute the warning disappears.

Now I can't tell if it is a bug not having the `type` attribute, I always thought `type="text"` is implicit for `<input>`.

Anyways, thanks for the help, I can live with that ;)

**Comment by ferllings** [comment]
Thanks, I can reproduce now.
